classes/Admin/Popups.php

```diff
@@ -562,6 +563,34 @@ public static function fields() {
 							),
 						),
 					),
+					'sound' => array(
+						'open_sound'   => array(
+							'label'    => __( 'Opening Sound', 'popup-maker' ),
+							'desc'     => __( 'Select a sound to play when the popup opens.', 'popup-maker' ),
+							'type'     => 'select',
+							'std'      => 'none',
+							'priority' => 10,
+							'options'  => array(
+								'none'         => __( 'None', 'popup-maker' ),
+								'beep.mp3'     => __( 'Beep', 'popup-maker' ),
+								'beep-two.mp3' => __( 'Beep 2', 'popup-maker' ),
+								'beep-up.mp3'  => __( 'Beep Up', 'popup-maker' ),
+								'chimes.mp3'   => __( 'Chimes', 'popup-maker' ),
+								'correct.mp3'  => __( 'Correct', 'popup-maker' ),
+								'custom'       => __( 'Custom Sound', 'popup-maker' ),
+							),
+						),
+						'custom_sound'   => array(
+							'label'    => __( 'Custom Sound URL', 'popup-maker' ),
+							'desc'     => __( 'Enter URL to sound file.', 'popup-maker' ),
+							'type'     => 'text',
+							'std'      => '',
+							'priority' => 10,
+							'dependencies' => array(
+								'open_sound' => array( 'custom' ),
```

Comment:
Just an FYI, not that you need to change it, you can also simplify in this case to `'open_sound' => 'custom',`

Reply:
Ah, I just followed what some of the other fields were doing. We probably need to whip up some docs about our fields system at some point.